Week-1/week1_assignment_1_2_cattyBoard/cattyBoard.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style>
+    * {
+      box-sizing: border-box;
+    }
+
+    header {
+      float: left;
+      display: flex;
+      background-color: #3a3b3c;
+      padding-left: 25px;
+      width: 100vw;
+      text-align: left;
+
+      color: white;
+    }
+
+    nav {
+      position: absolute;
+      float: left;
+      width: 3em;
+      height:85vh;
+      background: #a9a9a9;
+      padding: 10px;
+    }
+    li button {
+
+      border: none;
+      outline: none;
+      background: none;
+
+    }
+    p button {
+      padding-left:2em;
+      border: none;
+      outline: none;
+      background: none;
+
+    }
+    nav ul {
+      list-style-type: none;
+      padding: 0px;
+    }
+
+    article {
+      float:center;
+      padding: 80px 10px;
+      width: 80vw;
+      background-color: #fff;
+      height: 200px;
+      text-align: center;
+    }
+    table {
+             border-collapse: collapse;
+             width: 100vw;
+         }
+    td {
+      width:90vw
+      float:center;
+      align: center;
+      padding: 10px;
+    }
+    tr {
+      padding-left: 40px;
+    }
+    tr:nth-child(even) {
+              background-color: #a9a9a9;
+          }
+    section:after {
+      content: "";
+      display: table;
+      clear: both;
+    }
+
+    footer {
+      position: absolute;
+      bottom: 0px;
+      height:15vh;
+      width: 100vw;
+      background-color: #3971ac;
+      vertical-align: bottom;
+      padding: 20px;
+      text-align: center;
+      color: white;
+    }
+    li {
+
+      padding-top:4em;
+    }
+    .chartimg {
+      float: left;
+      padding: 2em;
```

Comment:
"rem" is better for most cases in my humble opinion... but Is giving 2em necessary here ?